sale_mail_templates/templates/sale_mail_templates.xml

```diff
@@ -1,42 +1,46 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <odoo>
+<!-- Some languages can't have same phrasing as English, so we
+separate quotation/order where it is used.-->
+<template id="doc_name">
+    <span t-if="is_quote">quotation</span>
+    <span t-else="">order</span>
+</template>
+
+<!-- Must copy/paste to be able to split it in translation file -->
+<template id="doc_name_proforma">
+    <span t-if="is_quote">quotation</span>
+    <span t-else="">order</span>
+</template>
 
-<!-- Adapt sale.email_template_edi_sale template -->
 <template
     id="mail_body_sale_send"
     name="Sales Order: Send by email"
 >
-    <t t-call="base_templates.mail_body_layout_basic">
-        <t t-call="base_templates.msg_partner_refer">
-            <t t-set="partner" t-value="record.partner_id"/>
-        </t>
-        <t t-call="base_templates.msg_doc_description_with_amount">
-            <t t-set="doc_text">
-                <span name="span_start_text">Here is</span>
-                <span
-                    name="span_proforma_text"
-                    t-if="ctx.get('proforma')"
-                >
-                    pro-forma invoice
-                </span>
-                <span
-                    name="span_quotation_text"
-                    t-elif="record.state in ('draft', 'sent')"
-                >
-                    the quotation
-                </span>
-                <span name="span_order_text" t-else="">the order</span>
+    <t t-set="is_quote" t-value="object.state in ('draft', 'sent')"/>
+    <div style="margin: 0px; padding: 0px;">
+        <p style="margin: 0px; padding: 0px; font-size: 13px;">
+            Hello,
+            <br/><br/>
+            Your
+            <t t-if="ctx.get('proforma')">
+                Pro forma invoice for <t t-call="sale_mail_templates.doc_name_proforma">quotation</t> <strong t-out="object.name or ''">S00052</strong>
+                <t t-if="object.origin">
+                    (with reference: <t t-out="object.origin or ''"></t> )
+                </t>
+                amounting in <strong t-out="format_amount(object.amount_total, object.pricelist_id.currency_id) or ''">$ 10.00</strong> is available.
+            </t>
+            <t t-else="">
+                <t t-call="sale_mail_templates.doc_name">quotation</t> <strong t-out="object.name or ''"></strong>
+                <t t-if="object.origin">
+                    (with reference: <t t-out="object.origin or ''">S00052</t> )
```

Comment:
Why demo data is used on templates?

Reply:
Its the way email qweb templates are written now. Its weird, but thats how Odoo decided to use it..